tsl/test/expected/cagg_errors.out

```diff
@@ -38,21 +38,22 @@ as
 select location, count(*) from conditions , mat_t1
 where conditions.location = mat_t1.c
 group by location WITH NO DATA;
-ERROR:  only one hypertable allowed in continuous aggregate view
+ERROR:  continuous aggregate view must include a valid time bucket function
```

Comment:
I have removed these from these files. Since the error cases are already covered in cagg_joins test, adding them here seems redundant. Please let me know if this sounds fine to you too.

Reply:
It's fine!